packages/terra-table/src/terra-dev-site/doc/table/reference-proptypes/sectionShape.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import rowShape from '../../../../proptypes/rowShape';
+import sectionHeaderShape from '../../../../proptypes/sectionHeaderShape';
+/**
+ * The props table parser continues to have trouble detecting the propTypes of a component that
+ * returns `null`. I've duplicated the propType definition here so that a table could be parsed for it.
+ */
+const propTypes = {
+  /**
+    * The children list items passed to the component.
+    */
+  rows: PropTypes.arrayOf(rowShape),
```

Comment:
I don't think this is rendering like you were hoping: 
![Screen Shot 2019-12-10 at 10 58 18 AM](https://user-images.githubusercontent.com/14099737/70561778-b2d72800-1b50-11ea-8b8b-70a869cc3df3.png)

Reply:
Dave doesn't really have a way of fixing this. We'd have to correct this in the props table loader and even then we're reliant on react-docgen. V5 might fix this but we'd have to test.